pulsar-transaction/coordinator/src/main/java/org/apache/pulsar/transaction/coordinator/impl/TxnLogBufferedWriter.java

```diff
@@ -217,9 +224,9 @@ public void asyncAddData(T data, AddDataCallback callback, Object ctx){
                 // Avoid missing callback, do failed callback when error occur before add data to the array.
                 int recordsCountAfter = dataArray.size();
                 if (recordsCountAfter == recordsCountBeforeAdd){
-                    callback.addFailed(new ManagedLedgerException.ManagedLedgerFencedException(e), ctx);
+                    callback.addFailed(new ManagedLedgerInterceptException(e), ctx);
```

Comment:
If you're notifying through the callback I'm not sure we actually need a log line in this case. I expect the callback to notify the log, no?